packaging_automation/citus_package.py

```diff
@@ -154,12 +156,21 @@ def get_signing_credentials(packaging_secret_key: str,
     return SigningCredentials(secret_key=secret_key, passphrase=passphrase)
 
 
-def write_postgres_versions_into_file(input_files_dir: str, package_version: str):
-    release_versions = get_supported_postgres_release_versions(f"{input_files_dir}/{POSTGRES_MATRIX_FILE_NAME}",
-                                                               package_version)
-    nightly_versions = get_supported_postgres_nightly_versions(f"{input_files_dir}/{POSTGRES_MATRIX_FILE_NAME}")
+def write_postgres_versions_into_file(input_files_dir: str, package_version: str, os_name: str = "",
+                                      platform: str = ""):
+    # In ADO pipelines function without os_name and platform is used. If these parameters are unset
```

Comment:
I do not really follow this comment. Why and how do we call this with missing parameters?